tests/models/test_dagbag.py

```diff
@@ -170,20 +172,20 @@ def my_flow():
             assert dagbag.import_errors[tf_2.name].startswith("Ignoring DAG")
             assert dagbag.dags == dags_in_bag  # Should not change.
 
-    def test_zip_skip_log(self):
+    def test_zip_skip_log(self, caplog):
         """
         test the loading of a DAG from within a zip file that skips another file because
         it doesn't have "airflow" and "DAG"
         """
-        with self.assertLogs() as cm:
-            test_zip_path = os.path.join(TEST_DAGS_FOLDER, "test_zip.zip")
-            dagbag = models.DagBag(dag_folder=test_zip_path, include_examples=False)
+        caplog.set_level(logging.INFO)
+        test_zip_path = os.path.join(TEST_DAGS_FOLDER, "test_zip.zip")
+        dagbag = models.DagBag(dag_folder=test_zip_path, include_examples=False)
 
-            assert dagbag.has_logged
-            assert (
-                f'INFO:airflow.models.dagbag.DagBag:File {test_zip_path}:file_no_airflow_dag.py '
-                'assumed to contain no DAGs. Skipping.' in cm.output
-            )
+        assert dagbag.has_logged
+        assert (
+            f'File {test_zip_path}:file_no_airflow_dag.py '
+            'assumed to contain no DAGs. Skipping.' in caplog.text
+        )
```

Comment:
I could not get `caplog` to log exactly like the `unittest`, so I stripped out the part that was not in order, and luckily it's not the important part

Reply:
Yeah, caplog behaves slightly differently. This test is good enough.